### ISSUE TITLE
Folder as workspace blinking

### ISSUE THREAD

**Issue by da411d** [deadem/notepad-pp-linter#18]
Folder-as-workspace blinkeing when linter is analyting because of temp file. It is annoying. Could you change temp file to some default temp folder, please?

![image](https://user-images.githubusercontent.com/12462454/84431855-ae411880-ac34-11ea-9644-1273698bdbcf.png)

**Comment by deadem** [comment]
Hmmm. Yep, this is ugly hack and I'm don't have any good solution. But I can try to add some config settings for temp directory and search the linter config in parent directories by myself.

**Comment by da411d** [comment]
@deadem still waiting for fix

**Comment by deadem** [comment]
@da411d I don't like the idea with temp folder and search config... This broke checking the dependencies. Does "Folder-as-workspace" show the hidden files? Maybe hidden attribute can solve the problem?

**Comment by da411d** [comment]
Just checked it, no:(
Folder-as-workspace displays "hidden" and "system" files as well

**Comment by da411d** [comment]
Maybe you can pass code using `stdin` instead of file?

**Comment by da411d** [comment]
Eslint supports it:
https://eslint.org/docs/user-guide/command-line-interface

**Comment by deadem** [comment]
Fixed in 700b00d2965c83e5c2ee715beb387b1acd20dba5

You can get update from https://github.com/deadem/notepad-pp-linter/tree/master/bin or https://sourceforge.net/projects/notepad-linter/

**Comment by deadem** [comment]
```xml
<linter stdin="1" extension=".js" command="C:\Users\deadem\AppData\Roaming\npm\eslint.cmd --stdin --format checkstyle"/>```
